docs/UserGuide.md

```diff
@@ -223,7 +298,11 @@ Supplier at index 3 is deleted, assuming it existed initially. Otherwise, an err
 TO UPDATE IMAGE AFTER FINAL UPDATE TO APPLICATION!!!
 ![delete command](images/deleteSupplierCommand.png)
 
-### Mark a supplier with a status : `mark -s`
+[Back to Table of Contents](#table-of-contents)
+
+---
+
+### 3.3.4 Mark a supplier with a status : `mark -s`
```

Comment:
were you planning to change the header of all commands to end with -ing, should this be "Marking a supplier ..." instead? same applies to the find and sort commands as well